infrastructure/parameters/prod.parameters.json

```diff
@@ -84,7 +84,7 @@
       "value": 2147483648
     },
     "maxStatsDbSizeBytes": {
-      "value": 697932185600
+      "value": 912680550400
```

Comment:
Not sure if this actually equates to a 300GB increase?

The old value of 697932185600 (650GB) + 322122547200 (300 GB) = 1020054732800

Reply:
Not too certain.  I've made this change though. To get to `912680550400 ` I used `1073741824 x 300` & then added the initial value of `650GB` in bytes to it.